src/mgmt/srv_query.c

```diff
@@ -68,7 +68,7 @@ ds_mgmt_bio_health_query(struct mgmt_bio_health *mbh, uuid_t dev_uuid,
 	if (uuid_is_null(dev_uuid) && strlen(tgt) == 0) {
 		/* Either dev uuid or tgt id needs to be specified for query */
 		D_ERROR("Neither dev_uuid or tgt_id specified for BIO query\n");
-		return -DER_INVAL;
+		return -1;
```

Comment:
Why change it from -DEV_INVAL to -1?

Reply:
not sure how this slipped by , I will change back